cli/Valet/PhpFpm.php

```diff
@@ -103,4 +103,20 @@ function fpmConfigPath()
 
         return $confLookup[$this->brew->linkedPhp()];
     }
+
+
+    /**
+     * Switch between versions of installed PHP
+     *
+     * @return void
+     */
+    function switchTo($version)
+    {
+        $version = preg_replace('/[.]/','',$version);
+        if (! $this->brew->installed('php'.$version)) {
+            throw new DomainException("This version of PHP not installed.");
+        }
+        $this->cli->passthru('brew unlink '.$this->brew->linkedPhp());
+        $this->cli->passthru('brew link php'.$version);
```

Comment:
Moved `PhpFpm::install()` to the `valet` file where I am already making calls to `PhpFpm`